### PR TITLE
PE-550: Quote the balance in AR instead of Winston

### PR DESCRIPTION
**Pull request by matibat**

### CODE REVIEW

File: src/commands/get_balance.ts [hunk 1]
```diff
@@ -2,14 +2,19 @@ import { cliWalletDao } from '..';
 import { CLICommand, ParametersHelper } from '../CLICommand';
 import { AddressParameter, SeedPhraseParameter, WalletFileParameter } from '../parameter_declarations';
 
+const anArInWinston = 1_000_000_000_000;
+const numberOfDigitsAfterComma = 8;
```
Comment (arielmelendez) [inline src/commands/get_balance.ts:6]: Why limit this to 8 digits? There's 10^12 Winston per AR. Therefore we need 12 decimal places for precision.

File: src/commands/get_balance.ts [hunk 2]
```diff
@@ -2,14 +2,19 @@ import { cliWalletDao } from '..';
...
 new CLICommand({
 	name: 'get-balance',
 	parameters: [WalletFileParameter, SeedPhraseParameter, AddressParameter],
 	async action(options) {
 		const parameters = new ParametersHelper(options);
 		const address = await parameters.getWalletAddress();
-		const balance = await cliWalletDao.getAddressWinstonBalance(address);
-		console.log(balance);
+		const balanceInWinston = await cliWalletDao.getAddressWinstonBalance(address);
+		const balanceInAR = balanceInWinston / anArInWinston;
```
Comment (arielmelendez) [inline src/commands/get_balance.ts:15]: Consider using `arweave.ar.winstonToAr` instead.

**Review by arielmelendez — CHANGES_REQUESTED** [review]
Number of decimal places is the critical change needed.

File: src/commands/get_balance.ts (hunk 2 above)
Comment (matibat) [reply src/commands/get_balance.ts]: Here it is: 84da1c34155b75342e49264d08cf55eaf9601b8b

**Pull request merged**